debug.log

```diff
@@ -0,0 +1 @@
+[0202/185024.265:ERROR:crash_report_database_win.cc(469)] failed to stat report
```

Comment:
We don't need this file in our repo :)

Reply:
Fixed: https://github.com/wsakolski/political-compass/pull/3/commits/58b7179ad085a87eff2b1720530f84811d0f4423